build.gradle

```diff
@@ -97,20 +97,78 @@ javafx {
 }
 
 def JLINK_DIR = "$buildDir/rcv"
+
+// Comma-separated list of modules
 // Necessary to explicitly include due to https://github.com/BrightSpots/rcv/issues/625
 def MODULES_TO_ADD = "com.ctc.wstx"
 
 // ### jlink plugin settings
 jlink {
     imageDir = file(JLINK_DIR)
     imageZip = file(JLINK_DIR + ".zip")
-    addOptions '--add-modules', MODULES_TO_ADD, '--strip-debug', '--compress', '2',
-            '--no-header-files', '--no-man-pages'
+    addOptions '--add-modules', MODULES_TO_ADD,
+               '--strip-debug', '--compress', '2',
+               '--no-header-files', '--no-man-pages'
     mergedModule {
         requires "java.xml"
     }
     launcher {
-        name = "rcv"
+        // TODO Sync version number with release.yml and Main.java (github.com/BrightSpots/rcv/issues/662)
+        name = "RCTab"
+        version = "1.3.1"
```

Comment:
I assume it was 1.3.1 just for testing purposes? Can we set to 1.4.0 now?

Reply:
I'm a little afraid to set it to 1.4.0 prematurely, as we'd have fake but signed `1.4.0` versions floating around. I guess the same is true with `1.3.1`?

The issue stems from the fact that Mac won't allow a bundle to have a `-alpha` at the end, so we need some way of signifying a fake/development version number. See: https://stackoverflow.com/questions/70066384/javafx-gradle-jlink-plugin-jpackageimage-fails-because-of-1-0-snapshot-conta

Perhaps we should use 0.0.0 unless we are building a release? Or maybe this just isn't a real issue, and we're okay having "false" 1.4.0s around?

Reply:
Dumb Apple. That may be a blessing in disguise in a small way, though, because we don't want to be creating these builds / releasing alpha versions. I'd say we set it to `1.4.0-alpha` here now so the find-replace is easier later (plus I guess it'll block building).